packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -401,6 +403,10 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
                 <RaiseHand displayType={options?.displayType} />
               )
           }
+          {
+            /* @conditional-compile-remove(reaction) */
+            <Reaction displayType={options?.displayType} />
```

Comment:
We should make sure that this button can be hidden through the CallControl options like the other buttons in the control bar.